paper/src/main/kotlin/ltd/redeye/vanguard/paper/VanguardPlugin.kt

```diff
@@ -39,6 +40,14 @@ class VanguardPlugin : JavaPlugin(), VanguardPlugin {
         vanguard = VanguardCore(this)
 
         server.pluginManager.registerEvents(PlayerPaperEvents(), this)
+
+        if (vanguard.config.sendAnonymousStats) {
+            initMetrics();
+        }
+    }
+
+    private fun initMetrics() {
```

Comment:
Make this a method on the `VanguardPlugin` and check the config there, so we're not needlessly doing this for each platform implementation